src/common/utils.js

```diff
@@ -352,6 +352,16 @@ export function cleanTxLogs(tx: EdgeTransaction) {
   return JSON.stringify(asCleanTxLogs(tx), null, 2)
 }
 
+// Convert number strings in scientific notation to decimal notation using biggystring
+export function biggyScience(num: string): string {
```

Comment:
You don't need this function, you can just do:
`parseFloat('1e10').toFixed(18)`

Reply:
We need a big num library because `Number.MAX_SAFE_INTEGER.toString().length < 18`.